content/docs/enterprise/branding/logo.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Logo
+description: Add your own logo and favicon to the pomerium console.
+---
+You may add a URL to link to your logo and favicon which will replace the Pomerium defaults.
+![Replace the Logo and Favicon in Pomerium Enterprise](img/svg_logo_console.png)
+
+It will also appear in the Open Source webpages.
```

Comment:
```suggestion
It will also appear for all the user facing webpages too.

```

In general, we call the open source components of Pomerium, Pomerium Core.  `s/Open Source/Core/g`